test/Dockerfile

```diff
@@ -1,22 +1,18 @@
-FROM python:3.5.2
+FROM hashicorp/terraform as terraform
 
-ENV TERRAFORM_VERSION 0.11.8
+FROM python:3.6
+
+ENV TERRAFORM_VERSION 0.12.16
 ENV NODEJS_VERSION 8
 
-RUN curl -sL https://deb.nodesource.com/setup_${NODEJS_VERSION}.x | bash - \
-    && apt-get install nodejs unzip \
-    && rm -rf /var/lib/apt/lists/* \
-    && npm install -g terraform-plan-parser \
-    && curl -sL https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip | \
-       funzip - > /usr/local/bin/terraform \
-    && chmod 755 /usr/local/bin/terraform \
-    && pip install --extra-index-url https://pip-test.techservices.illinois.edu/index/test \
+RUN pip install --extra-index-url https://pip-test.techservices.illinois.edu/index/test \
         sdg-test-behave-terraform
 
 COPY . /usr/local/src
+COPY --from=terraform /bin/terraform /usr/local/bin/
 
 WORKDIR /usr/local/src/test
 
 VOLUME /root/.aws
 
-ENTRYPOINT ["behave", "--stop", "--no-capture"]
+ENTRYPOINT ["behave", "--stop", "--no-capture", "-v", "-D", "log_level=DEBUG"]
```

Comment:
Do you think verbose should be the default? I suggest you have a more minimal entrypoint with the defaults set as command arguments like so:

CMD [ "--stop", "--no-capture", "-v", "-D", "log_level=DEBUG" ]
ENTRYPOINT [ "behave" ]

This allows the runner of the container to change the parameters like so:

```
docker run -it -v ~/.aws:/root/.aws:ro --rm terraform_modules_test --help
usage: behave [options] [ [DIR|FILE|FILE:LINE] ]+

Run a number of feature tests with behave.

positional arguments:
...

```
```

Reply:
I'll adjust the Dockerfile to reflect these suggestions. I originally set the verbose settings for debugging purposes.